en/releases/centreon-core.md

```diff
@@ -373,6 +373,26 @@ future.
 
 ## Centreon Broker
 
+### 21.04.6
+
+Release date: `null`
+
+#### Improvements
+
+- Improved the multiplexing of events, which was a performance bottleneck. The the processing speed of queued events should be significantly increased.
+
+#### Bug fixes
+
+- Fixed a regression due to the central broker's cache generation optimization, which was too thorough and prevented BAM from computing KPIs based on boolean rules
+- The central broker's cache generation loaded too much data and took too much time when BAM was activated.
+- Fixed an issue that could cause segmentation faults in centreon-engine when scheduling external commands
+- Fixed a design issue to prevent from trying to access variables of broker's new logger when the logger is stopped. This issue could cause segmentation faults.
```

Comment:
```suggestion
- Fixed a design issue to avoid trying to access variables of broker's new logger when the logger is stopped. This issue could cause segmentation faults.
```